appserver/payara-appserver-modules/microprofile/opentracing/src/main/java/fish/payara/microprofile/opentracing/cdi/TracedInterceptor.java

```diff
@@ -198,7 +197,7 @@ private String getOperationName(final InvocationContext invocationContext, final
             .getConfigOverrideValue(Traced.class, "operationName", invocationContext, String.class)
             .orElse(traced.operationName());
 
-        if (!"".equals(operationName)) {
+        if (operationName == null || !operationName.isEmpty()) {
```

Comment:
While this is faithful translation of original condition, I don't think operationName can be null, as it is result of `Optional.orElse(annotationAttribute with default "")`.

Reply:
Fixed, nice catch, I did not notice the Optional :)